core/tests/protractor/editorAndPlayer.js

```diff
@@ -26,9 +26,69 @@ var ExplorationPlayerPage =
 
 describe('Full exploration editor', function() {
   var explorationPlayerPage = null;
+  var parentId2 = null;
+  var refresherExplorationId = null;
+  var currentExplorationId = null;
+  var pathname = null;
+  var parentId1 = null;
 
   beforeAll(function() {
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    users.createUser('user4@editorAndPlayer.com', 'user4EditorAndPlayer');
+    users.login('user4@editorAndPlayer.com');
+
+    workflow.createAndPublishExploration('Title 1', 'Algebra',
+      'It is a sample exploration.');
+    browser.getCurrentUrl().then(function(url) {
+      pathname = url.split('/');
+      /* In the url a # is added at the end that is not part of exploration
+         ID */
+      parentId1 = pathname[4].slice(0, -1);
+    });
+
+    workflow.createAndPublishExploration('Title 2', 'Algebra',
+      'It is a sample exploration.');
+    browser.getCurrentUrl().then(function(url) {
```

Comment:
See if you can move this to editor.getExplorationIdFromUrl() -- it's a bit low-level for this file.

Reply:
There was an existing function that did this in general, used that.